cmd/network/verification.go

```diff
@@ -218,6 +219,12 @@ func (e *EgressVerification) setup(ctx context.Context) (*aws.Config, error) {
 		e.log.Debug(ctx, "cluster %s found from OCM: %s", e.ClusterId, cluster.ID())
 		e.cluster = cluster
 
+		if e.cluster.Hypershift().Enabled() {
```

Comment:
Small conversation about this here: https://redhat-internal.slack.com/archives/CMD1L32M9/p1717424955110269

We may want to additionally

1. Take that bash script and just make it a new osdctl command (for another PR, not here 😄 )
2. Leave comments here explaining _why_ HCPs aren't supported, and when we can remove this block of code

Reply:
Good suggestions thanks! Will add the info from the thread.